src/common/mfu_flist_walk.c

```diff
@@ -335,6 +335,7 @@ static void walk_readdir_process_dir(const char* dir, CIRCLE_handle* handle)
             struct stat st;
             mfu_file_t* mfu_file = *CURRENT_PFILE;
             int status = mfu_file_lstat(dir, &st, mfu_file);
+            // TODO handle status != 0
```

Comment:
Did you mean to delete this TODO?

Reply:
`status` is not used here, which gives a compile warning. But I think the proper fix is not to remove `status`, but to actually use/check it. At the very least, the returned `st` shouldn't be used if `status != 0`. I was trying to avoid mixing functional changes with compile warning fixes, and opted to just add a TODO (but the warning remains here).

Reply:
Oh, right.  I read that as a deletion, not an added line.  Thanks for adding that.